package.json

```diff
@@ -32,6 +32,7 @@
     "dist:svg:maki": "svg-sprite --symbol --symbol-dest . --shape-id-generator \"maki-%s\" --symbol-sprite dist/img/maki-sprite.svg node_modules/@mapbox/maki/icons/*.svg",
     "dist:svg:mapillary:signs": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/mapillary-sprite.svg node_modules/mapillary_sprite_source/package_signs/*.svg",
     "dist:svg:mapillary:objects": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/mapillary-object-sprite.svg node_modules/mapillary_sprite_source/package_objects/*.svg",
+    "dist:svg:rapid:objects": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/rapid-object-sprite.svg node_modules/mapillary_sprite_source/package_objects/*.svg",
```

Comment:
We discussed this already, but I'll flag this: we're creating a whole new file here when we really only need 4 of the sprites. @hieule21 feel free to respond with the task # of the tech debt task we talked about yesterday!

Reply:
changed `rapid-features` to `rapid-mapillary-features` and updated the affected files.

Reply:
We discussed this further on Discord and decided to go this way.